### ISSUE TITLE
Tooltip in app that explains on InformOfTooltip view why you can't copy the mnemonic - other apps may be able to read it

### ISSUE THREAD

**Issue by paulshapiro** [mymonero/mymonero-app-js#213]
For extra clarity - users may not realize they want to copy it til they're on the page

**Comment by paulshapiro** [comment]
Collab with Paul for copy

**Comment by apxs94** [comment]
@paulshapiro - I can foresee *some* scenarios where people might want to create a wallet quickly... and require the hand-writing of the mnemonic would hinder this.

Perhaps offer users a check box to say "I accept the risks of copying", or, a way to securely safe the mnemonic to a folder location in text format - such that quick creation of a wallet is possible?

I'm alright with push back on this idea - but as someone testing the wallet right now, writing the mnemonic is cumbersome.

**Comment by paulshapiro** [comment]
Yes exactly. There was another issue somewhere around here with discussion (incl suraeNoether) about a toggle in settings to enable copying. We can use the same or similar toggle to allow copy utility btns to be displayed in the WalletDetails InfoDisclosing view.

**Comment by apxs94** [comment]
Sounds great. I realized screenshotting the mnemonic was possible, so went that route for the test.

/tangent
In particular when f.lux kicks in (reduces blue light in evenings) am having some issues seeing text boxes and reading some of the grayer text on this black background. Has anyone else mentioned this? Perhaps a "light" GUI option will remedy this in the long run.

**Comment by paulshapiro** [comment]
Indeed. Light theme toggle in settings was one of the first things on my list. The main limiting factors are (lesser) I'd have to pop into Sketch and generate a few things and (major) that some UI components will need to observe the switch and requery. Very easy code but will take a little bit of time. I think there's an issue for light theme on this repo.

**Comment by apxs94** [comment]
Okay, excellent, thanks Paul! Looking good so far. Will keep playing with it.

**Comment by paulshapiro** [comment]
Just an update: tooltip could actually just say what the risk of copying it is - not that that would stop 99% of users from copying it before they know to read the tooltip.. so.. imperfect design solution.. still thinkin
